slemicro/create_vm_with_eib.sh

```diff
@@ -29,13 +32,17 @@ while getopts 'f:e:n:h' OPTION; do
 	esac
 done
 
+[ -z "${ENVFILE}" ] && { usage && die "\"-f <path/to/variables/file>\" required" 2;}
+[ -z "${EIBFOLDER}" ] && { usage && die "\"-e <path/to/eib/folder>\" required" 2;}
+
 set -a
 # Get the env file
 source ${ENVFILE:-${BASEDIR}/.env}
 # Some defaults just in case
 CPUS="${CPUS:-2}"
 MEMORY="${MEMORY:-2048}"
 VMNAME="${NAMEOPTION:-${VMNAME:-slemicro}}"
+MACADDRESS="${MACADDRESS:-null}"
```

Comment:
are you sure it could be null by default?

Reply:
It should be null because later it will fail if no mac is provided https://github.com/suse-edge/misc/pull/109/files#diff-1fe767c7e29ef34f346230be6794a5513ee46bdcc30a65713ccf78b29fdbddfdR72 (in fact this is wrong, it should be `== null`, I'll fix it)